Release/src/http/client/http_client_winhttp.cpp

```diff
@@ -892,12 +895,14 @@ class winhttp_client final : public _http_client_communicator
             return GetLastError();
         }
 
-        // Set timeouts.
-        int milliseconds = static_cast<int>(config.timeout<std::chrono::milliseconds>().count());
-        milliseconds = std::max<decltype(milliseconds)>(milliseconds, 1);
-        if (!WinHttpSetTimeouts(m_hSession, milliseconds, milliseconds, milliseconds, milliseconds))
         {
-            return GetLastError();
+            // Set timeouts.
+            int milliseconds = static_cast<int>(config.timeout<std::chrono::milliseconds>().count());
```

Comment:
```suggestion
            const int milliseconds = static_cast<int>(
                std::max(config.timeout<std::chrono::milliseconds>(), std::chrono::milliseconds{1}).count());
```

Reply:
That's nicer. (It wasn't my code, I just indented it.)